### PR TITLE
feat - added home page of ecommerce with responsiveness

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Created Landing Page Of Ecommerce .(Home.html)

-> Used Card and grid of component library

-> Tried to make it full responsive

### CODE REVIEW

File: header/header.css [hunk 1]
```diff
@@ -36,7 +37,7 @@
   height: 100vh;
   width: 80%;
   transition: all 0.3s ease;
-  z-index: 1;
```
Comment (lannisterrr) [inline header/header.css:39]: make z-index seperate classes like z-index-sm , z-index-lg etc.

**Review by lannisterrr — COMMENTED** [review]
Preview not working correctly

File: home/home.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+#radio3:checked ~ .navigation-auto .auto-btn3 {
+  background: var(--theme-color);
+}
+
+#radio4:checked ~ .navigation-auto .auto-btn4 {
+  background: var(--theme-color);
+}
+
```
Comment (prathmeshjagtap) [inline home/home.css:107]: all buttons  have same color so try to reduce the repeating code

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]

File: header/header.css (hunk 1 above)
Comment (ashishnnnnn) [reply header/header.css:39]: Done :+1:

File: home/home.css (hunk 2 above)
Comment (ashishnnnnn) [reply home/home.css:107]: Hii Prathmesh .. these audio button have same colour .. But all will not have the color at same time..
Only one of them is active when corresponding input is clicked ..
So to take that in account with css , i have used repetitive code .. 
Couldn't find any alternative of this except to handle these with javascript

Comment (prathmeshjagtap) [reply home/home.css:107]: oh ok then

**Review by prathmeshjagtap — APPROVED** [review]

File: home/home.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
```
Comment (shubhamkumar648) [inline home/home.html:7]: you can also use  script tag below the body

File: home/home.html [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+  </head>
+  <body>
+    <div class="navigation-bar">
+      <a class="hemburgur" href="#"><i class="fas fa-bars"></i></a>
+      <div class="side-hidden-bar hidden">
+        <div class="user-info pad-2 flex-align jst-sp-sb">
+          <img
+            class="avatar avatar-md"
+            src="../data/avatar/ashish.jpg"
+            alt=""
```
Comment (shubhamkumar648) [inline home/home.html]: give some name in alt attribute

File: home/home.html [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+      class="featured-head fnt-3 flex-center-column text-center fnt-w-500 gap-2"
+    >
+      <p>Featured Categories</p>
+      <div class="thick-bar"></div>
+    </div>
+    <div class="container">
+      <div class="responsive-grid">
+        <div class="card vertical-card">
+          <div class="text-img img-container">
+            <img
```
Comment (shubhamkumar648) [inline home/home.html:132]: add alt attribute

File: home/home.html [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+            />
+            <div class="text-overlay text-center flex-center-column">
+              <div class="title">Studio - Live Equipments</div>
+            </div>
+          </div>
+        </div>
+        <div class="card vertical-card">
+          <div class="text-img img-container">
+            <img
+              src="https://www.johnsmusic.in/uploads/johns-music/categories/trad-544023_s.png?1427392910"
```
Comment (shubhamkumar648) [inline home/home.html:167]: same as  well

**Review by shubhamkumar648 — APPROVED** [review]
you forgot to add alt attribut in image tag ! remaing looks good to me

File: home/home.html (hunk 3 above)
Comment (ashishnnnnn) [reply home/home.html:7]: Done :+1:

File: home/home.html (hunk 6 above)
Comment (ashishnnnnn) [reply home/home.html:167]: Done

File: home/home.html (hunk 5 above)
Comment (ashishnnnnn) [reply home/home.html:132]: Added

File: home/home.html (hunk 4 above)
Comment (ashishnnnnn) [reply home/home.html]: Added

**Pull request merged**